plugins/modules/onepassword_info.py

```diff
@@ -375,6 +381,7 @@ def main():
                 username=dict(type='str'),
                 master_password=dict(required=True, type='str', no_log=True),
                 secret_key=dict(type='str', no_log=True),
+                service_account_token=dict(type='str', no_log=True),
```

Comment:
I think this value isn't used by the module.

Reply:
i removed the service_account_token value from the module part.